### DISCUSSION TITLE
Usage hurdles (Arch Linux, QEMU/KVM)

### DISCUSSION THREAD

**Discussion by StarwolfDancing** [ChrisTitusTech/linutil#822] [General]
Hi guys,
thank you for your great work. It helps me at work and at home! Cool stuff!
I started to use `linutil` on a fresh Arch Linux (QEMU/KVM) installed with [archinstall](https://github.com/archlinux/archinstall).
- The first hurdle was that I had update the data base to get going. 
- Next `unzip` was unknown for installing the `Bash prompt`
After that, everything works great!

**Comment by koibtw** [comment]
Thanks for sharing your thoughts here. Issues you mentioned should be fixed. Closing this.
